packages/integration-tests/src/api/interaction.ts

```diff
@@ -106,6 +116,15 @@ export const createSocialAuthorizationUri = async (
     followRedirect: false,
   });
 
+export const sendPrepareTotp = async (cookie: string) =>
```

Comment:
this function name is confusing, seems it's used to init a backend totp pending verification. What about `initTotp(PendingVerification)` or something else that can make this instantly understandable?

Reply:
Updated